marketplace/netlify-build/README.md

```diff
@@ -0,0 +1,25 @@
+# Netlify Build and Preview
+
+This sidebar extension allows you to build and preview up to 3 Netlify sites.
```

Comment:
I'd be more descriptive, add something similar to: "...directly from your entry editor. It's also possible to navigate to the site with a single click"

Reply:
Added in more detail.